pod/perldelta.pod

```diff
@@ -171,6 +171,38 @@ On platforms that don't support Inf/NaN values in floating-point numbers (such
 as VAX), C<builtin::inf> and C<builtin::nan> now throw a runtime error (rather
 than breaking the perl build). [GH #22882]
 
+=item *
+
+L<ExtUtils::ParseXS> has been extensively refactored internally and
+extensive tests added. Most of these changes shouldn't be visible
+externally with a few exceptions, the main ones being:
+
+The generated C code, especially for returning values, may have changed
+slightly, and in some cases be slightly more efficient (in particular,
+using TARG more often to return a value rather than creating a new
+temporary).
+
+The parser is more likely give warnings now on XS errors which previously
+would have just silently generated invalid C code.
+
+One XS bug has been fixed in a way that may be highly visible. Previously
+when parsing the parameters of an XS sub declaration, if a parameter
+couldn't be parsed, it was quietly ignored. This meant that it would still
+consume an argument, but wouldn't declare a C variable: a bit like the
+perl-level C<my ($a, undef, $c) = @_>. Now, it gives a compile error. This
```

Comment:
```suggestion
Perl-level C<my ($a, undef, $c) = @_>. Now, it gives a compile error. This
```
Maybe? Following the convention that "Perl" refers to the language and "perl" to the implementation.